app/client/jsx/Room.jsx

```diff
@@ -221,14 +230,22 @@ class Room extends Component {
                     onClick={this.onSwitchRoom.bind(this)}
                     showMapButton={mapState.mapVisible}
                     showMapTooltip={mapState.showMapTooltip}
-                    handleOpenMap={this.handleOpenMap.bind(this)}></Navigation>
+                    handleOpenMap={this.handleOpenMap.bind(this)}
+                    handleOpenEvents={this.handleOpenEvents.bind(this)}></Navigation>
                 <Beforeunload onBeforeunload={this.handleBeforeUnload.bind(this)} />
                 <Modal
                     isOpen={this.state.showMap}
                     onAfterOpen={this.handleOpenMap.bind(this)}
                     onRequestClose={this.handleCloseMap.bind(this)}>
                         <Map onRoomClick={this.onSwitchRoom.bind(this)} handleCloseMap={this.handleCloseMap.bind(this)}></Map>
                 </Modal>
+                <Modal
+                    isOpen={this.state.showEvents}
+                    onAfterOpen={this.handleOpenEvents.bind(this)}
```

Comment:
am i reading this correctly that clicking the button triggers `handleOpenEvents` but we're also calling in `onAfterOpen`?

Reply:
Hm...yeah that doesn't make sense. I don't think we need that callback.

Reply:
FYI I removed this callback but something funky happens when I do a similar thing for the map modal. Will look into this later.